### PR TITLE
style: Update text and styling on settings page

### PR DESCRIPTION
**Pull request by joshlarson**
# Before
Screenshot 2023-11-06 at 12 01 34 PM

# After
Screenshot 2023-11-06 at 1 43 18 PM

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205651022494294

### CODE REVIEW

File: assets/src/components/settingsPage.tsx [hunk 1]
```diff
@@ -30,13 +27,10 @@ const SettingsPage = (): ReactElement<HTMLDivElement> => {
     <div className={`l-page c-settings-page ${mobileMenuClass}`}>
       <Notifications />
       <div className="c-page__container">
-        <h1 className="c-page__title">Settings</h1>
-
```
Comment (firestack) [inline assets/src/components/settingsPage.tsx:34]: The Figma doc is a unclear on this, but I'd check with design that they wanted the `Settings` heading removed

File: assets/src/components/settingsPage.tsx [hunk 2]
```diff
@@ -30,13 +27,10 @@ const SettingsPage = (): ReactElement<HTMLDivElement> => {
...
         <div className="c-page__section">
           <h2 className="c-settings-page__section-header">Vehicle Settings</h2>
           <ToggleSetting
-            icon={<MapIcon />}
-            label="Vehicle labels on map"
+	    label={<div>Vehicle labels on <strong>regular</strong> buses</div>}
```
Comment (firestack) [inline assets/src/components/settingsPage.tsx:33]: praise: nice choice on the semantics here 👍🏻

File: assets/src/components/settingsPage.tsx (hunk 1 above)
Comment (joshlarson) [reply assets/src/components/settingsPage.tsx:34]: Good call! I'll doublecheck

File: assets/src/components/settingsPage.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (joshlarson) [reply assets/src/components/settingsPage.tsx]: Love it - I didn't know that that was a thing I could do!

I messed around with the formatting, so I can't just click "commit suggestion", but I'm following up with that as we speak.

**Review by firestack — APPROVED** [review]
pending design review, this LGTM 👍🏻

**Pull request merged**